app/shared-components/PremiumPromoModal/PremiumPromoModal.scss

```diff
@@ -150,17 +151,17 @@ $condensed-font-family: Roboto Condensed, "Open Sans", "Helvetica Neue", Helveti
 }
 
 .PremiumPromoModal__text-link-container {
-	display: flex;
-	justify-content: space-evenly;
+  display: flex;
+  justify-content: space-evenly;
```

Comment:
@wlycdgr Is this ok for Edge?

Reply:
It is but it looks a little better with space-around. Pushing.

Reply:
...fixed.